apps/end-tenancy/acceptance/features/tenant-details.js

```diff
@@ -202,6 +227,22 @@ Scenario('I see the correct title if I am checking a second tenant', function *(
   I.see(tenantDetailsPage.name.content.checkAnother);
 });
 
+Scenario('I see the correct title if I am requesting an nldp for a second tenant', function *(
```

Comment:
maybe it's because I don't know the code-base but I couldn't work out where in here are you making a request for an nldp?